RecoMuon/GlobalTrackFinder/interface/GlobalMuonTrajectoryBuilder.h

```diff
@@ -48,5 +49,9 @@ class GlobalMuonTrajectoryBuilder : public GlobalTrajectoryBuilderBase {
   edm::InputTag theTkTrackLabel;
   edm::EDGetTokenT<reco::TrackCollection> allTrackerTracksToken;
   edm::Handle<reco::TrackCollection> allTrackerTracks;
+  edm::InputTag thePrimaryVtxLabel;
+  edm::EDGetTokenT<reco::VertexCollection> primaryVertexToken;
+  edm::Handle<reco::VertexCollection> vertexCollection;
+  bool selectHighPurity_;
```

Comment:
Maybe I am missing the logic, but what does `selectHighPurity_` is used for in this GlobalMuonTrajectoryBuilder?
Also the vertexCollection does not seem to be used anywhere here...

Reply:
was missing a line in GlobalMuonTrajectoryBuilder that is added back in latest commit